app/models/order_detail.rb

```diff
@@ -129,6 +129,13 @@ def self.for_facility_url(facility_url)
     details
   end
 
+  def self.purchased_active_reservations
+    where(state: ['new', 'inprocess'])
```

Comment:
Yeah, I agree. That was something I didn't really want to take on with this fix right now, especially as some of them would involve full refactoring of the old `scope`/`condition:` syntax.